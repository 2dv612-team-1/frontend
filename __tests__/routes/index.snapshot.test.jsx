import React from "react";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";
import Routes from "../../src/routes";

describe("Routes", () => {
  it("should render correctly", () => {
    const tree = shallow(<Routes />);
    expect(toJson(tree)).toMatchSnapshot();
  });
});
