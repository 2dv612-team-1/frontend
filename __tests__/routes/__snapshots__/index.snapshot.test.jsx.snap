// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Routes should render correctly 1`] = `
<div>
  <Route
    component={[Function]}
    exact={true}
    path="/"
  />
  <Route
    component={[Function]}
    path="/login"
  />
  <Route
    component={[Function]}
    path="/logout"
  />
  <Route
    component={[Function]}
    path="/register/:role"
  />
  <AuthRoute
    access="admin"
    role={null}
  >
    <Route
      component={[Function]}
      path="/companies"
    />
  </AuthRoute>
  <AuthRoute
    access="company"
    role={null}
  >
    <Route
      component={[Function]}
      path="/representatives"
    />
  </AuthRoute>
</div>
`;
